### ISSUE TITLE
No map background

### ISSUE THREAD

**Issue by cdkonecny** [Stefal/rtkbase#192]
**Describe the bug**
Everything seems to work except no background appears for map on web GUI.   Same whether maptiler key is used or not.
Zoom, pin, and crosshair are visible in grey field

**To Reproduce**
1. Go to http://192.168.*.**/status

**Expected behavior**
See street map 

**Software/Hardware (please complete the following information):**
 - RTKBase release number 2.3.1
 - Ubuntu 20.04.3 LTS
 - Raspberry Pi 4B 2GB
 - Ublox F9P]
 - Connection type: USB

**Screenshots**
![image](https://user-images.githubusercontent.com/15312751/151612849-7df8fc95-18e0-4896-a9cb-c6ee02a56c26.png)

**Additional context**
Have tried multiple machines/browsers.   Windows/ios/android

**Comment by Stefal** [comment]
Yes, the tile provider have some problems since a few months. I hope @HurdmanBegins will fix this soon. The next RTKBase release will include another Osm tile.

But if you have entered a Maptiler key in settings.conf, it should work. This is Maptiler, this morning on my own base:
![image](https://user-images.githubusercontent.com/8255984/151651561-7e4a6c5d-725c-45ef-aa6f-4da229aae8fc.png)

**Comment by cdkonecny** [comment]
I have the same result/output with and without maptiler.   Even with only openstreetmap, it is just grey.   Am I missing something?

**Comment by Stefal** [comment]
To use Maptiler, you should:
- get an api key from Maptiler
- enter this key in the file `settings.conf` : https://github.com/Stefal/rtkbase/blob/2f5e80987553ec3d3c05ea6adad5e29a2f99dd62/settings.conf.default#L22
- Select this layer (Aerial_Hybrid) on the map selector (top-right)

**Comment by cdkonecny** [comment]
I have done that.   My problem is that I can’t even get a regular map to work much less maptiler.
